HtmlRenderer/Renderer.cs

```diff
@@ -437,6 +410,8 @@ public void Dispose()
         }
 
         static bool initialized = false;
+        private static string CachePath = null;
+        private static readonly string[] CacheFolders = new string[] { "blob_storage", "Cache", "Code Cache", "GPUCache" };
```

Comment:
The goal is:

1. Delete all cache folders
2. Do not delete any storage folders that may contain configuration for an overaly

There are 8 folders:
1. blob_storage
   - This folder contains disk versions of `blob://` URLs if they're too large to keep in memory. I've never seen an overlay write to this but I figured it should be included just in case, and it doesn't hurt anything to clear it.
   - See also: https://chromium.googlesource.com/chromium/src/+/HEAD/storage/browser/blob/README.md
2. Cache
   - This folder contains traditional cached objects such as image files, cached HTML/CSS, etc. Stored in a SQLite database I believe.
3. Code Cache
   - I think this folder is supposed to be for wasm cached compilation information, but I'm not 100% sure. I've never seen it contain actual data.
4. databases
   - I believe this folder contains a master SQLite database which maps out various other database schemas. Deleting it would cause e.g. IndexedDB databases to no longer be associated properly, causing settings to be lost.
5. GPUCache
   - GPU cache for webgl and such, can safely be cleared.
6. IndexedDB
   - Obvious, contains configuration for mopimopi at least, possibly other overlays
7. Local Storage
   - Obvious, contains configuration for various overlays
8. Session Storage
   - Obvious, contains configuration for various overlays

That's, uhh, a lot of information to include in a comment.

Maybe something like `// Delete all folders which don't potentially contain overlay configuration information` would work as a comment?

Reply:
> There are 8 folders

This was really my question, in terms of are there other folders that may exist that don't exist for you and should be cleaned, or do you know this is all of them, or is this just "sufficient"?

> I've never seen an overlay write to this but I figured it should be included just in case, and it doesn't hurt anything to clear it.

I've got at least a folder in there, which implies that a blob was written but then cleaned up.  Blobs are used by other parts of the storage system, e.g. blobs are used by indexeddb explicitly if a user writes a blob to the database and also implicitly if the values used are very large because leveldb is bad at them: https://chromium.googlesource.com/chromium/src/+/master/content/browser/indexed_db/docs/README.md

I just can't remember if blob_storage is the folder they get written to these days, but blob_storage feels like "storage" and not "cache".  If this is a "clobber caches" and not a "clobber everything including things saved by the overlay" operation, then I think you should leave blob_storage.

Reply:
`blob_storage` always gets a folder, even on a fresh start with no overlays added.

But just in case I removed it, and added the comment.

Reply:
Sorry, to clarify, I meant "I have a blob_storage subfolder", which to me implied that a blob had been written to disk at some point.

Reply:
Yes. When CEF is first initialized, it creates a `OverlayPluginCache/blob_storage/<some GUID>` folder, regardless of if you have a blob to store or not.

Reply:
Got it.  Sorry for the confusion.